src/openms/source/ANALYSIS/OPENSWATH/OpenSwathWorkflow.cpp

```diff
@@ -492,7 +494,7 @@ namespace OpenMS
     const TransformationDescription& trafo,
     const ChromExtractParams & cp,
     const ChromExtractParams & cp_ms1,
-    const Param & feature_finder_param,
+    Param & feature_finder_param,
```

Comment:
not sure this is needed.
Can you make an internal copy, or does the user need to get an updated Param object?

Reply:
This is added for line 527 where use_ms1_ion_mobility is updated based on class parameters. Is there a more elegant way to do this?

Reply:
@cbielow please see comment above and let me know your thoughts

Reply:
@jcharkow it is not really elegant, is there another way?

however, you need to consider that this changes the parameters here permanently. Is that what we want, or should we use `Param feature_finder_param` to only operate on a copy? Do we need to pass the changed parameter object back to the caller?